### PR TITLE
Testing out a branch

### PR DESCRIPTION
**Pull request by kyleschmitz**
Trying out a new change, please review.

### CODE REVIEW

File: HelloWorld.java [hunk 1]
```diff
@@ -2,6 +2,7 @@
 {
 	public static void main(String[] args)
 	{
-		System.out.println("Hello everybod in the world!");
+		System.out.println("This is my feature branch.");
+		System.out.println("I think it's pretty cool.");
```
Comment (kyleschmitz) [inline HelloWorld.java]: I don't think you should say "cool", it's not very professional.

File: HelloWorld.java [hunk 2]
```diff
@@ -2,7 +2,7 @@
 {
 	public static void main(String[] args)
 	{
-		System.out.println("This is my feature branch.");
-		System.out.println("I think it's pretty cool.");
+		System.out.println("This is my feature branch. Here's a change.");
+		System.out.println("I think it's pretty neat.");
```
Comment (sztabson) [inline HelloWorld.java]: good change

**Comment by HRezaeiM** [comment]
yea comment is possible as well :D

**Comment by alexgithub3** [comment]
Cool

**Pull request merged**